dashboard/test/models/script_level_test.rb

```diff
@@ -197,14 +197,14 @@ class ExampleSolutionsTests < ActiveSupport::TestCase
     sl2 = create(:script_level, lesson: sl.lesson, script: sl.script)
 
     summary = sl.summarize
-    assert_match Regexp.new("^#{root_url.chomp('/')}/s/bogus-script-[0-9]+/lessons/1/levels/1$"), summary[:url]
+    assert_match Regexp.new("^#{root_url(protocol: '//').chomp('/')}/s/bogus-script-[0-9]+/lessons/1/levels/1$"), summary[:url]
```

Comment:
I would suggest using `CDO.studio_url` here too, and below:
```
[staging] dashboard > root_url(protocol: '//').chomp('/')
=> "//staging-studio.code.org"
[staging] dashboard > CDO.studio_url
=> "//staging-studio.code.org"
```

```suggestion
    assert_match Regexp.new("^#{CDO.studio_url}/s/bogus-script-[0-9]+/lessons/1/levels/1$"), summary[:url]
```

Reply:
Oh great catch!